era5_cloud/create_artifact.jl

```diff
@@ -80,13 +83,26 @@ function create_cloud(variables, outfile_path; small=false)
             attrib = attribs,
         )
         ncout[name][:,:,:,:] = reverse(ncin[name][begin:THINNING_FACTOR:end,begin:THINNING_FACTOR:end,:,begin:THINNING_FACTOR:end], dims=2)
+        if variable == "relative_humidity"
+            for i in 1:length(z)
+                for t in 1:length(time_)
+                    ncout[name][:,:,i,t] = clamp.(ncout[name][:,:,i,t], FT(0), FT(100)) / FT(100)
+                end
+            end
```

Comment:
Would it work if we just do `ncout[name] = clamp.(ncout[name], FT(0), FT(100)) / FT(100)` without the loop?